apps/windowsChromeDriver.py

```diff
@@ -0,0 +1,86 @@
+import subprocess
+import requests
+import os
+import zipfile
+
+def get_chrome_version():
+    """Get the Chrome browser version installed on Windows."""
+    try:
+        output = subprocess.check_output(
+            r'reg query "HKEY_CURRENT_USER\Software\Google\Chrome\BLBeacon" /v version',
+            shell=True,
+            encoding='utf-8'
+        )
+        # Extract version from the command output
+        version = output.split()[-1]
+        print(f"Chrome Version: {version}")
+        return version
+
+    except Exception as e:
+        print(f"Error getting Chrome version: {e}")
+        return None
+
+
+def get_chrome_driver_download_url(version):
```

Comment:
You are calling the version directly in this get_chrome_driver_download_url function and also you are doing the same while calling the functions at last. 
Please check and change the naming conventione not to call directly the vesion of the chrom.